include/range/v3/utility/concepts.hpp

```diff
@@ -249,14 +249,22 @@ namespace ranges
 
             struct Same
             {
-                template<typename T, typename U>
-                auto requires_(T &&, U &&) -> decltype(
+                template<typename ...Ts>
+                struct same : std::true_type {};
+                template<typename T, typename ...Us>
+                struct same<T, Us...> : meta::fast_and<std::is_same<T, Us>...> {};
```

Comment:
Thanks, I've needed this a couple of times and it was weird to use `Same` for two types and `meta` for the variadic case.